### ISSUE TITLE
fail for the red vertical bar (W3C Editor’s Draft)

### ISSUE THREAD

**Issue by jake-abma** [w3c/tr-design#137]
inherited from https://github.com/w3c/wcag21/issues/156#issuecomment-329954497

Looking at https://w3c.github.io/wcag21/guidelines/ there’s also a fail for the red vertical bar (W3C Editor’s Draft) at the left: https://www.w3.org/StyleSheets/TR/2016/logos/ED which has white text (#ffffff) on a red background with hex code: #ff1b00

Beside a fail for Color Contrast: SC 1.4.3 we also have a failure of Images of Text SC 1.4.5

Note: The image when scaling fastly looses sharpness as the image is very small, at https://www.w3.org/TR/UNDERSTANDING-WCAG20/ the image (when you still choose to use an image) is much bigger, staying sharper at zoom levels, see: https://www.w3.org/StyleSheets/TR/2016/logos/WG-Note

**Comment by tripu** [comment]
About colour contrast: should we make that red a bit darker to fix the issue?

About the scaling, [`w3.org/StyleSheets/TR/2016/logos/ED`](https://www.w3.org/StyleSheets/TR/2016/logos/ED) is a PNG image, while [`w3.org/StyleSheets/TR/2016/logos/WG-Note`](https://www.w3.org/StyleSheets/TR/2016/logos/WG-Note) is SVG. (See all images here: [`w3.org/StyleSheets/TR/2016/logos`](https://www.w3.org/StyleSheets/TR/2016/logos/).)

Perhaps we should aim at serving all those images as SVG by default at some point soon, ie. add the few logos that are missing an SVG version at the moment?

(In any case, and [as I mentioned a few days ago](https://github.com/w3c/tr-design/issues/134#issuecomment-350773366), those background images are there for aesthetics reasons mainly, I think: the information they contain is redundant and is always shown as text at the top of the spec. ie, no information or feature of the document would be missing if for some reason those images failed to load or to display altogether.)
